src/crawlee/static_content_crawler/__init__.py

```diff
@@ -0,0 +1,10 @@
+from ._static_content_crawler import StaticContentCrawler
+from ._static_content_parser import BlockedInfo, StaticContentParser
+from ._static_crawling_context import ParsedHttpCrawlingContext
+
+__all__ = [
+    'BlockedInfo',
```

Comment:
Is this just a return type? If we do not accept it anywhere as a user-facing input argument, I believe we should not expose it.

Reply:
It will be useful for anyone creating it's own crawler based on custom parser.